worker/provisioner/provisioner_test.go

```diff
@@ -585,6 +585,64 @@ func (s *ProvisionerSuite) TestPossibleTools(c *gc.C) {
 	)
 }
 
+var validCloudInitUserData = `packages:
```

Comment:
put packages on the next line and use [1:] at the end of the string

Reply:
done